app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTest.kt

```diff
@@ -339,13 +339,12 @@ class BloodPressureEntrySheetControllerTest {
   @Suppress("Unused")
   private fun `params for checking valid date input`(): List<DoNotSaveBpWithInvalidDateTestParams> {
     return listOf(
-        DoNotSaveBpWithInvalidDateTestParams(OpenAs.New(patientUuid), InvalidPattern),
-        DoNotSaveBpWithInvalidDateTestParams(OpenAs.Update(UUID.fromString("f6f27cad-8b82-461e-8b1e-e14c2ac63832")), InvalidPattern))
+        DoNotSaveBpWithInvalidDateTestParams(OpenAs.New(patientUuid)),
+        DoNotSaveBpWithInvalidDateTestParams(OpenAs.Update(UUID.fromString("f6f27cad-8b82-461e-8b1e-e14c2ac63832"))))
   }
 
   data class DoNotSaveBpWithInvalidDateTestParams(
```

Comment:
We can change this supplier to just return `List<OpenAs` now, and remove the params class.